src/orange/orangeinp/IntersectRegion.cc

```diff
@@ -1226,15 +1226,15 @@ Parallelepiped::Parallelepiped(Real3 const& half_projs,
 void Parallelepiped::build(IntersectSurfaceBuilder& insert_surface) const
 {
     // Cache trigonometric values
-    real_type sinth, costh, sinphi, cosphi, sinal, cosal;
-    sincos(theta_, &sinth, &costh);
-    sincos(phi_, &sinphi, &cosphi);
-    sincos(alpha_, &sinal, &cosal);
+    real_type sin_th, cos_th, sin_phi, cos_phi, sin_al, cos_al;
```

Comment:
Ok but I wouldn't consider it a typo.

Reply:
Yeah I changed that one to avoid a false positive.